### PR TITLE
Hans main dev

### PR DESCRIPTION
**Pull request by hansbrenna**
Really just to check what I've changed

### CODE REVIEW

**Comment by hansbrenna** [comment]
@jgliss I don't understand why the test_varinfo test fails for conctc. Can you help?

File: pyaerocom/web/aerocom_evaluation.py [hunk 1]
```diff
@@ -849,8 +854,17 @@ def find_coldata_files(self, model_name, obs_name, var_name=None):
...
                             pass
+                        try:
+                            add_var_name = self.model_config[model_name]['model_add_vars'][var_name]
+                        except:
+                            add_var_name = None
                         if not m['var_name'] == var_name:
-                            match = False
+                            if isinstance(add_var_name,list):
+                                if not m['var_name'] in add_var_name:
+                                    match = False
```
Comment (jgliss) [inline pyaerocom/web/aerocom_evaluation.py:864]: Could you add a test that reaches this code (i.e. handling of model_add_vars as list)?

**Review by jgliss — CHANGES_REQUESTED** [review]
@hansbrenna I went through it mostly looks good, however, there are a few things that we need to clarify / fix before merging. The most pressing issue is probably the molmass of nh3, which might be wrong. And also the units related issues, where I would appreciate if you could make adjustments in variable names and / or units, if possible. Best if you comment on my individual comments and we take it from there.

File: pyaerocom/molmasses.py [hunk 2]
```diff
@@ -10,6 +10,8 @@
              'so4'      : 96.06,
              'no'       : 30.01,
              'no2'      : 46.0055,
+             'hno3'     : 63.01,
+             'nh3'      : 43.028,
```
[the remark this answers is not in the document]
Comment (jgliss) [reply pyaerocom/molmasses.py:14]:
```suggestion
             'nh3'      : 17.031,
```

**Comment by jgliss** [comment]
@hansbrenna thanks for the meeting! As discussed, maybe we can use the following variable conventions to distinguish the units:

`concno3` -> ug m-3
`concNno3` -> ug N m-3

And for variable names representing a certain PM size regime, we add pmX at the end, e.g. 

`concno3pm10`
`concno3pm25`

Just let me know when you are ready with the updates so I can revise

File: pyaerocom/data/variables.ini [hunk 3]
```diff
@@ -871,6 +871,17 @@ maximum = 1
...
+var_name = precip
+description = precipitation
+standard_name = dummy
+var_type = Meteo / Thermodynamics
+unit = mm
+minimum = 0
+maximum = 10000
+dimension = time,lat,lon
+
+
```
[the remark this answers is not in the document]
Comment (hansbrenna) [reply pyaerocom/data/variables.ini:884]: removing this.

File: pyaerocom/io/test/test_read_ebas.py [hunk 4]
```diff
@@ -73,11 +74,28 @@ class TestReadEbas(object):
...
+                         'concNtnh'
+                         'conccoc25',
                          'concss',
                          'concnh3',
                          'concno3',
+                         'concno310',
+                         'concno325',
                          'concnh4',
                          'conchno3',
+                         'concss10',
```
Comment (jgliss) [inline pyaerocom/io/test/test_read_ebas.py:97]:
```suggestion
                         'concsspm10',
```

File: pyaerocom/io/test/test_read_ebas.py [hunk 5]
```diff
@@ -73,11 +74,28 @@ class TestReadEbas(object):
...
+                         'concss25',
```
Comment (jgliss) [inline pyaerocom/io/test/test_read_ebas.py:98]:
```suggestion
                         'concsspm25',
```

**Review by jgliss — CHANGES_REQUESTED** [review]
Looks good! Just some minor re-iteration needed for variable defs. Seems also that the EBAS test PROVIDES_VARIABLES has some old variables in it (e.g. concec, concec25). Should be quick to fix for you.

**Comment by hansbrenna** [comment]
I've done the updates suggested. Waiting for the checks to run.

**Comment by hansbrenna** [comment]
Should we merge this now @jgliss ?

**Review by jgliss — APPROVED** [review]
Looks good now :)

**Pull request merged**